examples/material-next-app-router-ts/src/app/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import ThemeRegistry from '@/components/Theme/ThemeRegistry/ThemeRegistry';
+
+export const metadata = {
+  title: 'Next App with MUI5',
+  description: 'next app with mui5',
+};
+
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <html lang="en">
+      <body>
```

Comment:
Does MUI work with `next/font`?
```tsx
import {Inter} from 'next/font/google';

const inter = Inter({subsets: ['latin']});

// ...
<body className={inter.className}>
...
</body>
```
If it does, would be really helpful to include it in the example.

Reply:
Yes it works with the Next fonts, I can include it in the example.

Reply:
with next/fonts: Next.js automatically generates a styles file and inserts the fonts reference to it.

<img width="1225" alt="font_styles" src="https://github.com/mui/material-ui/assets/17010033/22d5cf2c-086c-4493-9011-8f999bc31d7d">


<img width="1234" alt="font" src="https://github.com/mui/material-ui/assets/17010033/954ae0cc-75d4-47a6-8e4c-6160792a34ca">